tests/e2e/performance/test_pvc_attachtime.py

```diff
@@ -106,18 +108,55 @@ def init_full_results(self, full_results):
         return full_results
 
     @pytest.fixture()
-    def pod(self, interface, pod_factory, pvc_factory):
+    def pod_obj_list(
+        self, interface, storageclass_factory, pod_factory, pvc_factory, samples_num
+    ):
         """
-        Prepare pod for the test
+        Prepare sample pods for the test
 
         Returns:
-            pod obj: Pod instance
+            pod obj: List of pod instances
 
         """
         self.interface = interface
-        pvc_obj = pvc_factory(interface=interface, size=self.pvc_size)
-        pod_obj = pod_factory(pvc=pvc_obj)
-        return pod_obj
+        pod_result_list = []
+
+        pvc_size = 5
+        self.msg_prefix = f"Interface: {self.interface}, PVC size: {pvc_size}."
+        # for i in range(samples_num):
+        #    pvc_obj = pvc_factory(interface=interface, size=pvc_size)
+        #    pod_obj = pod_factory(pvc=pvc_obj)
+        #    pod_result_list.append(pod_obj)
+
+        if self.interface == constants.CEPHBLOCKPOOL_THICK:
+            self.sc_obj = storageclass_factory(
+                interface=constants.CEPHBLOCKPOOL,
+                new_rbd_pool=True,
```

Comment:
Just wondering what the reason is to use a new rbd-pool? It is not required for thick-provisioning, any existing rbd-pool can be used.

Reply:
I think that would be convenient for the teardown. Even if a new pool is not mandatory, it is convenient for the testing.